assets/models/system/CodeLlama-13b-Python-hf/spec.yaml

```diff
@@ -5,10 +5,18 @@ properties:
   SHA: 488587ade99c84ea4325e6d620d307372728cc97
   inference-recommended-sku: Standard_NC24s_v3, Standard_NC24rs_v3, Standard_ND40rs_v2,
     Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
```

Comment:
batch is tested with1p-preview model and evaluation with preview-test1. For evaluation were we needed to test with preview-test1?

Reply:
Should be okay, as both version have been tested